gcloud/streaming/test_transfer.py

```diff
@@ -565,8 +565,8 @@ def test_GetRange_not_initialized(self):
     def test_GetRange_wo_total_size_complete(self):
         from six.moves import http_client
         from gcloud._testing import _Monkey
-        from gcloud._apitools import transfer as MUT
-        CONTENT = 'ABCDEFGHIJ'
+        from gcloud.streaming import transfer as MUT
+        CONTENT = b'ABCDEFGHIJ'
```

Comment:
I'm seeing maybe the `b''` before wasn't so out of place, but still am curious as to the why.